scripts/cc-cli/src/commands/collectCoins.ts

```diff
@@ -20,7 +18,19 @@ export function makeCollectCoinsCmd() {
     .description("Swap GCRE for CTC")
     .addOption(externalAddressOpt)
     .addOption(burnTxHashOpt)
-    .action(collectCoinsAction);
+    .action(collectCoinsActionSync);
+}
+
+function collectCoinsActionSync(options: OptionValues) {
+  collectCoinsAction(options)
+    .then(() => {
+      console.log("Success");
```

Comment:
```suggestion
      console.log(chalk.green("Success!"));
```

as in the removed `handleSuccess`.